lms/djangoapps/grades/models.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from xmodule_django.models import BlockTypeKeyField, CourseKeyField, UsageKeyField
+
+class Grade(models.Model):
+    """
+    A django model tracking persistent grades at the subsection level.
+    """
+
+    class Meta(object):
+        index_together = [
+            ['user_id', 'course_id', 'content_type', 'is_valid'],
+            ['course_id', 'updated'],
+            ['updated'],
+        ]
+
+    # TODO: timestamp discussions - do we need all three of [created, updated, subtree_edited_date]?
+    created = models.DateTimeField('creation timestamp', auto_now_add=True)
+    updated = models.DateTimeField('latest updated timestamp', auto_now=True)
```

Comment:
Could we use the `TimeStampedModel` for this instead? `from model_utils.models import TimeStampedModel`